docs/learn/learn-comparison-ethereum-2.md

```diff
@@ -42,6 +42,11 @@ the shards will implement the eWasm interface, finally making the system usable
 The network will also have "side chains" to interact with chains that are not under the finality
 protocol of Ethereum 2.0.
 
+> **Update**: The Ethereum 2.0 development roadmap has changed from the phases detailed above. 
```

Comment:
Instead of having this as a note, it may be better to have it flow with the original content. 

The "original" and what it phase(s) look like now.